x-pack/plugins/apm/scripts/test/api.js

```diff
@@ -82,7 +82,7 @@ if (server) {
 const cmd = [
   'node',
   ...(inspect ? ['--inspect-brk'] : []),
-  `../../../../scripts/${ftrScript}`,
+  `../../../../../scripts/${ftrScript}`,
```

Comment:
How did this break? Was this file moved into a subfolder or why is the extra `..` needed?